packages/prediction-app/src/features/send-chap-data/SendChapData.tsx

```diff
@@ -32,6 +32,20 @@ interface SendChapDataProps {
     onDrawerSubmit: () => void
 }
 
+const safeCreateFixedPeriodFromPeriodId = (
+    params: Parameters<typeof createFixedPeriodFromPeriodId>[0]
+) => {
+    try {
+        return createFixedPeriodFromPeriodId(params)
```

Comment:
This throws an error if it couldn't be found. The cause of this should be fixed by using correct period format below, however we don't want to crash the app if this fails - so I kept it in.